libs/components/forms/src/lib/modules/radio/fixtures/radio.component.fixture.ts

```diff
@@ -23,13 +23,17 @@ export class SkyRadioTestComponent implements AfterViewInit {
   @ViewChild(SkyRadioComponent)
   public checkboxComponent!: SkyRadioComponent;
 
-  public ngAfterViewInit() {
-    this.checkboxComponent.disabledChange.subscribe((value) => {
-      this.onDisabledChange(value);
-    });
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public onCheckedChange(checked: boolean): void {
+    /* */
   }
 
-  public onDisabledChange(value: boolean): void {}
+  public onClick(): void {
+    /* */
+  }
 
-  public onClick() {}
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public onDisabledChange(disabled: boolean): void {
```

Comment:
You can remove the `disabled` parameter to avoid the ESLint comment.

Reply:
I removed the parameter based on your feedback. I got in the habit of including parameters in the callback function because Angular's strict template checking throws errors when the template implementation doesn't match the class's implementation (e.g. `onDisabledChange($event)` and `onDisabledChange(): void`). Since we're not running strict template checking on the fixtures, it'll be okay to remove the unused parameters.